picard/tagger.py

```diff
@@ -1086,29 +1099,42 @@ def main(localedir=None, autoupdate=True):
     signal.signal(signal.SIGINT, signal.SIG_DFL)
 
     picard_args, unparsed_args = process_picard_args()
+
     if picard_args.version:
         return version()
     if picard_args.long_version:
         return longversion()
 
-    try:
-        from PyQt5.QtDBus import QDBusConnection
-        dbus = QDBusConnection.sessionBus()
-        dbus.registerService(PICARD_APP_ID)
-    except ImportError:
-        pass
-
-    tagger = Tagger(picard_args, unparsed_args, localedir, autoupdate)
+    should_start = picard_args.stand_alone_instance
 
-    # Initialize Qt default translations
-    translator = QtCore.QTranslator()
-    locale = QtCore.QLocale()
-    translation_path = QtCore.QLibraryInfo.location(QtCore.QLibraryInfo.LibraryLocation.TranslationsPath)
-    log.debug("Looking for Qt locale %s in %s", locale.name(), translation_path)
-    if translator.load(locale, "qtbase_", directory=translation_path):
-        tagger.installTranslator(translator)
+    if not should_start:
+        pipe_handler = pipe.Pipe(app_name=PICARD_APP_NAME, app_version=PICARD_FANCY_VERSION_STR, args=picard_args.FILE)
+        should_start = True in [pipe_handler.permission_error_happened, pipe_handler.is_pipe_owner]
```

Comment:
I've run a few benchmarks for 2 elements and `True in` seems to be way faster

https://pastebin.com/WXx8EaD4

Reply:
```suggestion
        should_start = True in {pipe_handler.permission_error_happened, pipe_handler.is_pipe_owner}
```

Use a set instead of a list.